### ISSUE TITLE
Keine Verbindung möglich

### ISSUE THREAD

**Issue by philro86** [TA2k/ioBroker.ford#11]
Ich bekomme mit dem Adapter keiner Verbindung. 
Das Log gibt nur folgende Fehlermeldungen aus: "Error: Request failed with status code 400" und "{"error_description":"CSIAQ0172E Der Erteilungstyp [password] wird nicht unterstützt. Unterstützte Erteilungstypen sind [authorization_code, refresh_token].","error":"unsupported_grant_type"}"

Mailadresse und Passwort sind korrekt. Woran kann es liegen?

**Comment by TA2k** [comment]
Bitte auf 0.0.12 updaten

**Comment by ali86x** [comment]
Update auf 0.0.12 ist nicht möglich trotz Beta-Repository.

Fehler kommen bei mir auch

**Comment by TA2k** [comment]
v0.1.1 hat ein neuen status fetch und eine 12v Überwachung bei unter 12.2 stellt sich der adapter ab.

die neue API liefert enorm viele Daten

**Comment by weindler** [comment]
> v0.1.1 hat ein neuen status fetch und eine 12v Überwachung bei unter 12.2 stellt sich der adapter ab.
> 
> die neue API liefert enorm viele Daten

@TA2k keine aktualisierte Daten mit der 0.1.1 außer neue Datenpunkte habe dann das Debug aktiviert, aber ich kann leider die Log Datei vom iobroker nicht runterladen, da sofort zur Ford Login Seite weitergeleitet wird.

**Comment by weindler** [comment]
Log ist per Mail unterwegs zu dir, als Debug Version.

**Comment by ali86x** [comment]
Ich Krieg das Log auch nicht vernünftig runtergeladen. Kann zwar nen Screenshot machen, aber das nützt wahrscheinlich nix.

**Comment by weindler** [comment]
> Ich Krieg das Log auch nicht vernünftig runtergeladen. Kann zwar nen Screenshot machen, aber das nützt wahrscheinlich nix.

ja, da stimmt was nicht, da muß @TA2k nochmal ran, es schaut aus als ob sich da der ford adapter in den admin adapter einklinkt und dann kommt hinter dem log ?page=login oder so ähnlich, ich habs über winscp direkt vom rechner runtergeladen dann geht es.

**Comment by TA2k** [comment]
Welche Datenpunkten aktualisieren sich nicht. 
Hast du "Update bei jedem interval" aktiv. 
Oder führst du remote.statusupdate aus

**Comment by weindler** [comment]
Update bei jedem Intervall ist bei mir aus und den remote Status habe ich auch noch nicht ausgeführt, muss man das jetzt machen?  Das geht aber massiv auf die Batterie dann kann man höchstens 2 mal am Tag aktualisieren, ging ja vorher auch und in der app sind ja aktuelle daten

**Comment by TA2k** [comment]
Welches Datum steht in der App wenn du runter ziehst

**Comment by weindler** [comment]
> Welches Datum steht in der App wenn du runter ziehst

Aktuell vor 6 Stunden also am 19.9. Gegen 6uhr 50

**Comment by TA2k** [comment]
steht das auch in den timestamps im adapter?

**Comment by weindler** [comment]
Jetzt klappt es, aber da haben sich ja alle Datenpunkte geändert oder sehe ich das falsch?

**Comment by weindler** [comment]
Log ist unauffällig, bisher keinen Fehler, danke dir für deine Arbeit.
